README.md

```diff
@@ -22,3 +22,36 @@ or using a browser: `http://localhost:9090`
 
 Please refer to the [lorax-composer](https://github.com/weldr/lorax)'s documenation as osbuild-composer is a drop-in replacement.
 
+## High-level overview
+
+![overview](docs/osbuild-composer.svg)
+
+### Frontends
+
+`osbuild-composer` is meant to be used with 3 different front-ends. The primary one, which is meant for general use, is cockpit-composer. It is part of the Cockpit project and unless you have a strong reason not to use it, you should use it. `composer-cli` is a command line tool that can be used with `osbuild-composer`.
```

Comment:
Maybe change to "`osbuild-composer` has two frontends:" (rather than 3, no need to talk about RCM yet).